backend/src/common/plugins/socket/socket.plugin.ts

```diff
@@ -0,0 +1,43 @@
+import { type FastifyInstance, type FastifyPluginCallback } from 'fastify';
+import { Server } from 'socket.io';
+
+import { SocketEvent } from './enums/enums.js';
+import { type ChatMessageGetAllItemResponseDto } from './types/types.js';
+
+const socket: FastifyPluginCallback = (
+    fastify: FastifyInstance,
+    _options,
+    done,
+) => {
+    const io = new Server(fastify.server, {
+        cors: {
+            origin: '*',
+            credentials: true,
+        },
+    });
+
+    io.on(SocketEvent.CONNECTION, (socket) => {
```

Comment:
io.of(SocketNamespace.CHAT).on(SocketEvent.CONNECTION, (socket) => {

Reply:
Fixed